feat: Additional MQTT commands

robertbeal opened the pull request:
No changes of note, more trying to help out given I work on the API's :) So an introduction and a reference to copy from

- I've dropped username + password from the Token. They should be disposed of rather than retained
  - if you reach out to platform-team@yotoplay.com we can get you your own ClientId
  - username/password authentication we'll eventually want to move away API consumers from. Mainly as the user has no way to revoke the access. And instead use a "on behalf of" model where a refresh token is issued to a requesting app at the user's approval.
- Some general tidy up, f-strings, code reduction etc... although it's been several years since I did any python so take my changes with a pinch of salt and personal preference

- I've added some missing MQTT commands and tried to simplify them as much as possible
- I've included links to our (work in progress) API documentation which may help

*I'm a long-time HA user so will try and help out on that repo at some point

cdnninja commented:
Amazing! Thank you so much for this.  Greatly appreciate this.  I am also very open to any feedback or concerns from Yoto. 

I am just an IT guy tinkering.  Not a dev by any stretch.

cdnninja commented:
Could I bug you for details on the colors? I was working on the constants for colors but found two different ways they are represented in the API and also seem inconsistent with testing to understand them.  The hex being pretty clear but the player API with items like: 0x602d3c being less clear. I think that number is Lilac but not certain.

robertbeal commented:
I've added an extra comment above RGB. It's just the RBG 0-255 values. It's not a part I've really worked on so not aware of how it's represented elsewhere.

Added a bluetooth command too. Apologies, I should have branched on my side and need to get my pyenv working so haven't been able to run linting.
